src/Jackett.Common/Indexers/DarmoweTorrenty.cs

```diff
@@ -0,0 +1,232 @@
+            var seedsMatch = _SeedsRegex.Match(detailsRow.TextContent);
+            var leechersMatch = _LeechersRegex.Match(detailsRow.TextContent);
+            var dateMatch = _DateRegex.Match(detailsRow.TextContent);
+            var sizeMatch = _SizeRegex.Match(detailsRow.TextContent);
+            var date = DateTime.MinValue; // In case of parsing failure
+            if (dateMatch.Success)
+            {
+                date = DateTime.ParseExact(
+                    $"{dateMatch.Groups[1].Value} +01:00", "dd/MM/yyyy zzz", CultureInfo.InvariantCulture);
+            }
+            var details = titleRow.QuerySelector("a[href^=\"details.php?id=\"]:has(span)");
+            var detailsLink = new Uri(SiteLink + details.GetAttribute("href"));
+            var encodedDownloadLink = detailsRow.QuerySelector("a[id^=\"download_\"]").GetAttribute("data-href");
+            var siteDownloadLink =  new Uri(SiteLink + Uri.UnescapeDataString(StringUtil.FromBase64(encodedDownloadLink)));
+            var hash = HttpUtility.ParseQueryString(siteDownloadLink.Query)["id"];
+            var magnet =
+                $"magnet:?xt=urn:btih:{hash}&tr=udp://tracker.opentrackr.org:1337&tr=udp://tracker.coppersurfer.tk:6969&tr=udp://tracker.internetwarriors.net:1337&tr=udp://tracker.leechers-paradise.org:6969";
+            var downloadLink = new Uri(magnet);
+            var bannerLink = detailsRow.QuerySelector("img[src^=\"./imgtorrent/\"]")?.GetAttribute("src");
+            var seeders = seedsMatch.Success ? int.Parse(seedsMatch.Groups[1].Value) : 0;
+            var leechers = leechersMatch.Success ? int.Parse(leechersMatch.Groups[1].Value) : 0;
+            var peers = seeders + leechers;
+            var release = new ReleaseInfo
+            {
+                Title = details.TextContent,
+                Category = categories,
+                Seeders = seeders,
+                BannerUrl = !string.IsNullOrEmpty(bannerLink) ? new Uri(SiteLink + bannerLink) : null,
+                Peers = peers,
+                PublishDate = date,
+                DownloadVolumeFactor = 0,
+                UploadVolumeFactor = 1,
+                Link = downloadLink,
+                Guid = detailsLink,
+                Comments = detailsLink,
+                Size = sizeMatch.Success ? ReleaseInfo.GetBytes(sizeMatch.Groups[1].Value) : 0
+            };
+            return release;
+        }
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var releases = new List<ReleaseInfo>();
+
+            var searchString = query.GetQueryString();
+            var searchUrl = BrowseUrl;
+            var queryCollection = new NameValueCollection
+            {
+                {"search", searchString},
+                {"category", "0"}, // multi category search not supported
+                {"erotyka", "1"}
+            };
+            searchUrl += "?" + queryCollection.GetQueryString();
+
```

Comment:
nearly done ;-)

search for `Dziewiąta Devyataya` yields no result with indexer but returns results on web site
url generated by:
indexer: https://darmowe-torenty.pl/torrenty.php?search=Dziewi%C4%85ta+Devyataya&category=0&erotyka=1
web site: https://darmowe-torenty.pl/torrenty.php?search=Dziewi%B1ta+Devyataya&category=0&erotyka=1

something to do with how the indexer is managing diacritics?

Reply:
nice catch, fixed by generating the query string using the site's preferred encoding.